Documentation/runtime-configuration.md

```diff
@@ -8,40 +8,37 @@ Reconfiguration requests can only be processed when the the majority of the clus
 
 ## Reconfiguration Use Cases
 
-Let us walk through the four use cases for re-configuring a cluster: replacing a member, increasing or decreasing cluster size, and restarting a cluster from a majority failure.
+Let us walk through some common reasons for reconfiguring a cluster. Most of these just involve combinations of adding or removing a member, which are explained below under [Cluster Reconfiguration Operations](#cluster-reconfiguration-operations).
 
-### Replace a Non-recoverable Member
+### Replace A Failed Machine
 
-The most common use case of cluster reconfiguration is to replace a member because of a permanent failure of the existing member: for example, hardware failure or data directory corruption.
-It is important to replace failed members as soon as the failure is detected.
-If etcd falls below a simple majority of members it can no longer accept writes: e.g. in a 3 member cluster the loss of two members will cause writes to fail and the cluster to stop operating.
+If a machine fails due to hardware failure, data directory corruption, or some other fatal situation, it should be replaced as soon as possible. Machines that have failed but haven't been removed adversely affect your quorum and reduce the tolerance for an additional failure.
 
-If you want to migrate a running member to another machine, please refer [member migration section][member migration].
+To replace the machine, follow the instructions for [removing the member][remove member] from the cluster, and then [add a new member][add member] in its place. If your cluster holds more than 50MB, it is recommended to [migrate the failed member's data directory][member migration] if you can still access it.
```

Comment:
We could, but I wanted to separate out based on whether its reactive (replace) or proactive (cycle). This way we can talk about how important it is to replace the machine quickly for replacement, but thats not needed for proactive, since you know you're reducing quorum willingly.